### PR TITLE
fix(FEC-9079): USER_SEEKED event triggering twice

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

ACT: event dispatch twice on touchend and click, click & touchend fired and doing the same thing for touch devices.
SOLUTION: handle it only in touchend and fix the getTime for touch devices.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/seekbar/seekbar.js [hunk 1]
```diff
@@ -99,28 +97,6 @@ class SeekBarControl extends Component {
...
-   * @param {Event} e - onClick event
-   * @returns {void}
-   * @memberof SeekBarControl
-   */
-  onTap(e: Event): void {
-    if (!this.props.isMobile) {
-      return;
-    }
-    const oldTime = this.props.player.currentTime;
-    const newTime = this.getTime(e);
```
Comment (OrenMe) [inline src/components/seekbar/seekbar.js]: won;t this kill the tap action? we support both dragging and tapping the seekbar. Did you check?

Comment (Yuvalke) [reply src/components/seekbar/seekbar.js]: Yes

Comment (OrenMe) [reply src/components/seekbar/seekbar.js]: Yes on which one of the questions? :-)

Comment (Yuvalke) [reply src/components/seekbar/seekbar.js]: Lol, of course I checked. It's support dragging, the problem was that on click and ontouchend did same thing..

**Pull request merged**